cmd/lotus-miner/sectors.go

```diff
@@ -1020,10 +1024,16 @@ var sectorsRenewCmd = &cli.Command{
 		for l, exts := range extensions {
 			for newExp, numbers := range exts {
 				scount += len(numbers)
-				addrSectors, err := policy.GetAddressedSectorsMax(nv)
-				if err != nil {
-					return err
+				var addrSectors int
+				if cctx.Int("max-sectors") == 0 {
+					addrSectors, err = policy.GetAddressedSectorsMax(nv)
+					if err != nil {
+						return err
+					}
+				} else {
+					addrSectors = cctx.Int("max-sectors")
```

Comment:
Nit: Could check that this is lower than policy.GetAddressedSectorsMax, and error if it's not.